src/h2load.cc

```diff
@@ -1709,6 +1852,9 @@ time for request: )" << std::setw(10) << util::format_duration(ts.request.min)
             << util::format_duration(ts.ttfb.sd) << std::setw(9)
             << util::dtos(ts.ttfb.within_sd) << "%" << std::endl;
 
+  if (config.conn_inactivity_timeout > 0 || config.conn_active_timeout > 0) {
+    std::cout << R"(requests timed out: )" << stats.req_timedout << std::endl;
+  }
```

Comment:
Rather than showing this in separate line, it would be more streamlined to print them after "requests: ..., N errored" line (e.g., "requests: ..., N errored, M timeout"), since this is a subset of failure, just like "errored."  What do you think about this?